tron/core/actionrun.py

```diff
@@ -653,6 +653,47 @@ def submit_command(self):
         mesos_cluster.submit(task)
         return task
 
+    def recover(self):
+        if self.mesos_task_id is None:
+            log.error(f'No task ID, cannot recover {self}')
```

Comment:
I think we never get here because of recovery.py:32. (due to `None` filtering)
What do you think about removing the `None` filter so that this code does get invoked and logs the "why"?

Reply:
Yeah, that makes sense. Will do.